app/helpers/entities_helper.rb

```diff
@@ -0,0 +1,17 @@
+module EntitiesHelper
+  def entities_total(group)
+    GroupEntity.where("group_id = #{group.id}").joins(:entity).sum('amount')
+  end
```

Comment:
[Optional]
- [ ] You should avoid making SQL queries outside of models. It makes it harder to test and debug your app.